ScnSoft-Education/LeetCodeTraining/LongestCommonPrefix13/LongestCommonPrefix.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace LeetCodeTraining.LongestCommonPrefix13
+{
+    public class Solution
+    {
+        public string LongestCommonPrefix(string[] inputStrings)
+        {
+            if (inputStrings == null || inputStrings.Length == 0)
```

Comment:
you can use a short form to check this expression like:
inputStrings?.Length == 0